shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Prtcl.hs

```diff
@@ -216,15 +229,15 @@ prtlSeqChecks ::
   WithOrigin (LastAppliedBlock crypto) ->
   BHeader crypto ->
   m ()
-prtlSeqChecks lab bh =
+prtlSeqChecks lab currentBh =
```

Comment:
More descriptive names are great!  In general, though, we've tried to keep the same names between the spec and the code. And sometimes when there are lots of long names in the spec it gets hard to read. It's all a  balancing act! I certainly don't want to slow you down, though!

Reply:
That's fair enough, I didn't realize the variable names were chosen to match the spec, I'll revert this change.